test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -40,10 +46,13 @@ public class TrustedTestCert<T> : IDisposable
 
         private bool _isDisposed;
 
+        private string _systemTrustedCertPath;
+
         public TrustedTestCert(T source,
             Func<T, X509Certificate2> getCert,
-            StoreName storeName = StoreName.TrustedPeople,
-            StoreLocation storeLocation = StoreLocation.CurrentUser,
+            StoreName storeName,
+            StoreLocation storeLocation,
+            TestDirectory dir,
```

Comment:
`dir` is used to store trusted certs in MacOS. To avoid creating a directory for each cert is best for it to be passed by the test.